examples/js/postprocessing/CubeTexturePass.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @author bhouston / http://clara.io/
+ */
+
+THREE.CubeTexturePass = function ( scene, camera, envMap, opacity ) {
+
+	THREE.Pass.call( this );
+
+	this.scene = scene;
```

Comment:
It isn't being used, I was just following the convention of many Passes that take "scene, camera" as their first two parameters and remember they them.  I'll remove it.